install-unix.sh

```diff
@@ -25,40 +31,62 @@ array_contains () {
     return $in
 }
 
+function install_vagrant() {
+    echo "installing for event_name=$event_name"
+    echo "If you have any issues, please send your install.log to code@magfest.org"
+
+    local event_name=$1
+    local vm_cmd="cd ~/uber/ && ./run-simple-deploy.sh $event_name && rm -f ./run-simple-deploy.sh"
+
+    git clone https://github.com/magfest/ubersystem-deploy
+    cd ubersystem-deploy
+    vagrant up
+    cp ../installfiles/run-simple-deploy.sh .
+    vagrant ssh -c "$vm_cmd"
+}
+
+function post_install() {
+    # not sure how / if we should do this on *nix
+    # on windows, can do 'start http://localhost:8000/uber/accounts/insert_test_admin' to open web browser
+
+    testadminString="Create your test user at http://localhost:8000/uber/accounts/insert_test_admin"
+
+    case "$OSTYPE" in
+      darwin*)  open "http://localhost:8000/uber/accounts/insert_test_admin"  ;;
+      solaris*) echo $testadminString ;;
+      linux*)   echo $testadminString ;;
+      bsd*)     echo $testadminString ;;
+      *)        echo $testadminString ;;
+    esac
+}
+
 if [ -z "$1" ]; then
+  echo "ERROR: you must specify which event you want to deploy."
   usage
   exit -1
 fi
+event_name=$1
 
-array_contains valid_eventnames $1 && valid_event=1 || valid_event=0
-if [ $valid_event -eq 0 ]; then 
+array_contains valid_eventnames $event_name && valid_event=1 || valid_event=0
+if [ ${valid_event} -eq 0 ]; then
+  echo "ERROR: $event_name is not a valid event name."
   usage
   exit -1
 fi
 
-# ----------------------------------------
-# meat starts here
-# ----------------------------------------
-echo installling event_name=$1
-
-vm_cmd="cd ~/uber/ && ./run-simple-deploy.sh $1 && rm -f ./run-simple-deploy.sh"
-git clone https://github.com/magfest/ubersystem-deploy
-cd ubersystem-deploy
-vagrant up
-cp ../installfiles/run-simple-deploy.sh .
-vagrant ssh -c "$vm_cmd"
+if [ -d "ubersystem_deploy" ]; then
```

Comment:
woops, yea Rob you're right this should be ubersystem-deploy.  i was testing it by creating the dir manually.  i r dumb